.php_cs

```diff
@@ -26,43 +26,11 @@ if (file_exists(__DIR__.'/local.php_cs')) {
 }
 
 return Symfony\CS\Config\Config::create()
+    ->level(Symfony\CS\FixerInterface::SYMFONY_LEVEL)
```

Comment:
This was the default in 1.x, so adding these doesn't actually change anything in terms of behaviour, but will make a difference when ever you come to use v2 of php-cs-fixer because the default has changed from symfony to psr2.